<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormPlotData</class>
 <widget class="QWidget" name="FormPlotData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>562</width>
    <height>799</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PlotData</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,1,0">
   <item row="2" column="0">
    <widget class="QScrollArea" name="scrollAreaColumnChoose">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>542</width>
        <height>68</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QCheckBox" name="cBoxV14">
         <property name="text">
          <string>v14</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cBoxV24">
         <property name="text">
          <string>v24</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cBoxRaw14">
         <property name="text">
          <string>raw14</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cBoxRaw24">
         <property name="text">
          <string>raw24</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>542</width>
        <height>611</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QTableView" name="table"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QCheckBox" name="checkBoxChooseGroup">
       <property name="text">
        <string>choose group</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditTarget"/>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelGo">
       <property name="text">
        <string>Go</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditGo"/>
     </item>
     <item>
      <widget class="QLabel" name="labelStatus">
       <property name="text">
        <string>0/0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tBtnPrev">
       <property name="text">
        <string>prev</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tBtnNext">
       <property name="text">
        <string>next</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QCheckBox" name="checkBoxHighlightMinMax">
       <property name="text">
        <string>highlight MinMax</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelMinMaxInfo">
       <property name="text">
        <string>Min Max info</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
